### PR TITLE
Navigation Component

### PR DESCRIPTION
**Pull request by Arnav-Menon**
first commit, pretty ugly ngl

### CODE REVIEW

File: components/Navigation.jsx [hunk 1]
```diff
@@ -1,7 +1,13 @@
 import React from "react";
 
 const Navigation = () => {
-  return <div>Navigation</div>;
+  return (
+    <div>
+      <img src={"../public/vercel.svg"} alt="acm@ucr" style={{float:"left"}}/>
+      <h3 style={{float:"left", paddingLeft:"2%"}}>member portal</h3>
+      <h4 style={{float:"right", wordSpacing:"15px", paddingRight:"2%"}}>profile forms resources "general"</h4>
```
Comment (kninja99) [inline components/Navigation.jsx:8]: For styles we are using tailwind so try to avoid using inline css

File: components/Navigation.jsx [hunk 2]
```diff
@@ -1,7 +1,13 @@
 import React from "react";
 
 const Navigation = () => {
-  return <div>Navigation</div>;
+  return (
+    <div>
+      <img src={"../public/vercel.svg"} alt="acm@ucr" style={{float:"left"}}/>
```
Comment (kninja99) [inline components/Navigation.jsx:6]: Use the Image component from next.js

**Review by kninja99 — CHANGES_REQUESTED** [review]
Left comments in sections that need to be changed. Haven't loaded branch on my end to look but try to upload images so we can see how it looks.

##### Things to address

* for css styling use tailwind
*  change img element to be a Image component from next.js

**Review by shahdivyank — COMMENTED** [review]
Make sure to pass the CI checks as well: 

`npm run format` and `npm run eslint` will help you format the code and get you logs for the linting errors you have

**Comment by shahdivyank** [comment]
Make sure to attach the issue to this pr as well

**Comment by Arnav-Menon** [comment]
Screenshot (4)

**Comment by Arnav-Menon** [comment]
thanks for the comments guys. a couple of notes from me, I couldn't get the acm text to work for me, configuring the className with "text-acm-black" and without gives me the same results in the screenshot. Also I wasn't able to find the below image for the logo, which should go to the left of "member portal" 
Screenshot_20221006_010043

**Review by shahdivyank — APPROVED** [review]

**Review by RayyanZaid — APPROVED** [review]

**Review by kninja99 — APPROVED** [review]

**Pull request merged**